.github/workflows/package-dependencies-update.yml

```diff
@@ -29,58 +29,61 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '14' # Node version should be 14.
+          check-latest: true
 
       - name: Validate installed node version
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
           echo "Using node version: $(node --version)"
           echo "Using npm version: $(npm -v)"
 
       - name: Install node modules package dependencies
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
           npm install
 
       - name: Verify outdated node modules package dependencies versions
-        continue-on-error: true # Continuing on error, we need to check for outdated modules, so it's okay for this step to fail.
+        continue-on-error: true # Continuing on error. The initial command will return an exit 1 code; however, this is okay.
         run: |
           echo "Preparing to list outdated node modules versions within current build"
           npm outdated
 
       - name: Update node modules package dependencies versions within package.json
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
+          mkdir -p .github/logs
```

Comment:
Creating a logs directory that will contain the package dependencies that were updated.